.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
Can we try making this `bionic` instead of 14.04?

Reply:
I can try xenial. But we are not there for bionic yet.
https://travis-ci.community/t/openjdk8-on-bionic/4373/5